### DISCUSSION TITLE
Ridondanza delle Notifiche di Trascrizione per le Riconciliazioni.

### DISCUSSION THREAD

**Discussion by gcuci** [italia/ansc#950] [Miglioramenti] [answered]
Riportiamo di seguito una segnalazione del Comune di Carpi:

Buongiorno,

segnaliamo una ridondanza nelle Notifiche che vengono prodotte, successivamente alla validazione/firma di un atto di riconciliazione (IdUseCase  350000).
L’evento non modifica lo stato civile degli sposi e di conseguenza non ha effetti da un punto di vista anagrafico; non dovrebbe essere prodotta la notifica per Trascrizione.

In attesa di riscontro,

cordiali saluti.

**Comment by gbuemi** [comment, marked as answer]
Buongiorno @gcuci,
confermiamo che per il caso d'uso indicato è corretto che vengano prodotte due notifiche ANSC, una per intestatario. Approvando la notifica anche solo per un singolo intestatario si innesca il flusso di notifica ANPR.
Per ottenere una notifica sui sistemi ANPR, è inoltre necessario comunicare la data di validità dell'atto tramite R014 /evento/data-validita/{version}  prima o dopo la validazione/firma dell'atto. 
Infatti, in fase di firma usc, se la data validità non è presente, viene restituito un warning con la seguente descrizione: _Nel caso in cui l'evento ha una risultanza anagrafica, ricordati che devi inserire la data di validità_.

Un saluto,
Supporto ANSC
